client/layout/masterbar/style.scss

```diff
@@ -719,6 +733,13 @@ body.is-mobile-app-view {
 			--color-masterbar-highlight: var(--color-sidebar-menu-selected-text);
 			--color-masterbar-item-active-background: var(--color-sidebar-background);
 
+			&.masterbar__item-my-sites {
+				border-top-left-radius: 0;
+			}
```

Comment:
<img width="241" alt="Screenshot 2024-08-02 at 11 07 19 AM" src="https://github.com/user-attachments/assets/c149cda1-4342-4d48-bbfc-a62dfa9384eb">

We probably want to keep this rounded on mobile.

Reply:
Nice catch, I'll fix it

Reply:
Fixed [here](https://github.com/Automattic/wp-calypso/pull/93213/commits/df004f9c4e81fb5a0e36366612a12bd3838a7889)